agent/Agent.js

```diff
@@ -374,6 +378,22 @@ class Agent extends EventEmitter {
     this.emit('unmount', id);
     this.ids.delete(component);
   }
+
+  _onScroll() {
+    this._requestScrollUpdate();
+  }
+
+  _updateScroll() {
+    this._scrollUpdate = false;
+    this.emit('refreshMultiOverlay');
+  }
+
+  _requestScrollUpdate() {
+    if (!this._scrollUpdate) {
+      window.requestAnimationFrame(this._updateScroll.bind(this));
```

Comment:
this will generate a new function at each call, better to store the *bound* function as a property (eg. `this._updateScroll = this._updateScroll.bind(this);` inside the constructor)